src/ralph/admin/mixins.py

```diff
@@ -128,6 +128,14 @@ def __init__(self, *args, **kwargs):
         else:
             self.change_views = copy(self.change_views) or []
         super().__init__(*args, **kwargs)
+        # create copy (subclass) of view class to assign unique
```

Comment:
if you need to comment it like that - you probably need separate function with nice name